src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -3248,6 +3244,7 @@
     <EmbeddedResource Include="NHSpecificTest\NH1291AnonExample\Mappings.hbm.xml" />
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="NHSpecificTest\NH3985\Mappings.hbm.xml" />
```

Comment:
Removed this and other SubType elements, however it seems to have been an ongoing issue with Visual Studio since VS2005, so there's no clear way to eradicate them once and for all. They appear to be completely harmless though.